app/Entities/Place/Place.php

```diff
@@ -124,4 +124,9 @@ public function localization()
     {
         return $this->hasMany(PlaceTranslation::class);
     }
+
+    public function avgRating()
```

Comment:
I was a little mistaken if we have RatingRepository then create method getAverageByPlace(int $placeId) and call that method from action. Because such $this->ratings you retrive collection of models, then on collection you count average. Would be better like this  $this->rating()->avg('rating'), but the best solution is using a repository